app/inpututils.cpp

```diff
@@ -270,6 +270,40 @@ bool InputUtils::cpDir( const QString &srcPath, const QString &dstPath )
   return true;
 }
 
+QString InputUtils::renameFile( const QString &srcPath, const QString &prefix )
```

Comment:
I would suggest to have time also as an argument to this function. The default value could be invalid time (`QDateTime()`). With invalid time we would use current time. This way it's easier to write unit tests for such function (deterministic).